qemu/tests/cfg/balloon_service.cfg

```diff
@@ -27,7 +27,7 @@
     polling_interval = 2
     polling_sleep_time = 10
     free_mem_cmd = wmic os get FreePhysicalMemory
-    ratio = 0.5
```

Comment:
"balloon_check" should use ratio that declared at balloon_check.cfg 

[root@localhost tp-qemu]# cat qemu/tests/cfg/balloon_check.cfg 
- balloon_check: install setup image_copy unattended_install.cdrom
    virt_test_type = qemu
    no Win2000, Fedora.8, Fedora.9, Fedora.10, RHEL.3, RHEL.4, Unix, livecd
    type = balloon_check
    balloon = balloon0
    balloon_dev_devid = balloon0
    balloon_dev_add_bus = yes
    iterations = 5
    free_mem_cmd = cat /proc/meminfo |grep MemFree
    Linux:
        ratio = 1
    Windows:
        ratio = 0.5